package.json

```diff
@@ -17,6 +17,8 @@
     "test:ci": "jest --ci --coverage --maxWorkers=2",
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
+    "cypress:run:html": "cypress run --spec './cypress/integration/html-reader/*'",
+    "cypress:run:pdf": "cypress run --spec './cypress/integration/pdf-reader/*'",
```

Comment:
Is there a reason to split these out or should we just run all of them all the time?

Reply:
hmmm, yea we don't need to separate these, it's good to run all the tests. My intention was to separate them and make debugging quicker when writing tests for html/pdf.